Helper/PaymentMethods.php

```diff
@@ -153,6 +153,11 @@ public function togglePaymentMethodsActivation(?bool $isActive =null): array
         }
 
         foreach ($this->getAdyenPaymentMethods() as $paymentMethod) {
+            //skip POS from toggling as this config is visual via Admin panel
+            if ($paymentMethod == Config::XML_ADYEN_POS_CLOUD) {
```

Comment:
This skip case should be extended with [EXCLUDED_PAYMENT_METHODS](https://github.com/Adyen/adyen-magento2/blob/5e5bdd63e00974c9ee2edc9f9bdfdd096e5eaba5/Model/Config/Backend/PaymentMethodsStatus.php#L35) const array. I think the usage of this array was skipped on [this](https://github.com/Adyen/adyen-magento2/pull/2476) PR.

Reply:
```suggestion
            if (in_array($paymentMethod, PaymentMethodsStatus::EXCLUDED_PAYMENT_METHODS]) {
```